pkg/ibmvpc/sg_analysis.go

```diff
@@ -272,11 +277,16 @@ func (sga *SGAnalyzer) getSGrules(sgObj *vpc1.SecurityGroup) (ingressRules, egre
 	return ingressRules, egressRules, nil
 }
 
+type ruleTarget struct {
+	cidr   *ipblock.IPBlock
+	sgName string // target specified is SG
+}
+
 type SGRule struct {
-	target      *ipblock.IPBlock
+	remote      ruleTarget
 	connections *connection.Set
 	index       int // index of original rule in *vpc1.SecurityGroup.Rules
-	local       *ipblock.IPBlock
+	local       ruleTarget
```

Comment:
local can remain of type `IPBlock`, it cannot reference another SG

Reply:
right. restored

Reply:
forgot to push ?

Reply:
No, weird. It was pushed.